internal/stream/stream.go

```diff
@@ -184,9 +187,38 @@ func (s *Stream) StartReader(reader Reader) {
 
 	sr.start()
 
-	for _, sm := range s.streamMedias {
+	for m, sm := range s.streamMedias {
 		for _, sf := range sm.formats {
 			sf.startReader(sr)
+			if m.Type == "video" {
```

Comment:
isn't `m` just an integer? ie the loop counter?

Reply:
No, it's a `*description.Media` because `streamMedias` is a map!

Reply:
You made me realize I shall do `m.Type == description.MediaTypeVideo`.
Thanks.